WBooks/StartingScreen/StartingScreenZeroController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  StartingViewController.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 22/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class StartingScreenZeroController: UIViewController {
```

Comment:
Just to have and ordered code, try to keep init functions first, then loadView, then viewController's life functions (like `viewDidLoad), and at the end your custom methods